packages/fx-core/src/plugins/resource/function/utils/depsChecker/funcPluginAdapter.ts

```diff
@@ -76,8 +77,8 @@ class FuncPluginAdapter implements IDepsAdapter {
     throw new Error("Method not implemented.");
   }
 
-  public setFeatureFlag(answers?: ConfigMap): void {
-    this.enabled = answers?.getBoolean(this.answerKey) || false;
+  public setFeatureFlag(answers?: Inputs): void {
+    this.enabled = answers![this.answerKey] as boolean || false;
```

Comment:
this looks dangerous, is there a better/safer way for this change?

Reply:
Its a workaround, after API refactor for resource plugins, answers will never be optional.

Reply:
If there will be API refactor in the future, why don't we remove all changes from resource plugin and integrate new API in the future? It will keeps our code is always stable instead of having many workaround.